VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -124,25 +121,29 @@ public void onCheckedChanged(RadioGroup group, int checkedId) {
 
         getTopicList();
 
-        ViewCompat.setBackgroundTintList(topicsSpinner, getResources().getColorStateList(R.color.edx_grayscale_neutral_dark));
-
         topicsSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
-
             @Override
             public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
-                // if a top-level topic is selected, go back to previous selected position
-                DiscussionTopicDepth topic = allTopicsWithDepth.get(position);
-                if (topic.getDepth() == 0) {
-                    topicsSpinner.setSelection(selectedTopicIndex);
-                    Toast.makeText(container.getContext(), "Top level topic cannot be selected.", Toast.LENGTH_SHORT).show();
-                } else
+                // Even though we disabled topics that aren't supposed to be selected, Android still allows you to select them using keyboard or finger-dragging
```

Comment:
FYI, this issue is discussed at https://groups.google.com/forum/#!topic/android-developers/9pUfM0CmB1o